### DISCUSSION TITLE
when setting up meson to compile i3 a dependency is missing even though I had installed it through both apt and homebrew

### DISCUSSION THREAD

**Discussion by Nuclear-Regeneration-inc** [i3/i3#6118] [Q&A]
the errror is  `meson.build:308:12: ERROR: Dependency lookup for libstartup-notification-1.0 with method 'pkgconfig' failed: Pkg-config for machine host machine not found. Giving up.` so I setup both libstartup and pkgconfig through apt and homebrew.

**Comment by orestisfl** [comment]
You need libstartup-notification0-dev

**Reply by Nuclear-Regeneration-inc** [reply]
I installed that but I still get the same error
